llm_on_ray/inference/predictors/hpu_predictor.py

```diff
@@ -196,6 +200,8 @@ def generate(self, input: GenerateInput, **config) -> GenerateOutput:
 
         self._process_config(config)
 
+        # TODO: Maybe we should get realtime load info of all cards, set a heathy usage ratio and pick the usable cards for serving.
+        #       So that some errors like OOM can be prevented, and the server will be more robust.
         if self.infer_conf.deepspeed:
             return ray.get(
                 [worker.generate.remote(prompt, **config) for worker in self.deepspeed_workers]
```

Comment:
Instead of using the fixed work, maybe we should spread the load to all cards when deepspeed is enabled.

Reply:
There is no card scheduling in Ray, we only start worker process from Ray. Card scheduling should be taken care of in the underlining hpu deepspeed/transformer code. Pls let me know if you find the scheduling is incorrect.